biomaj_core/utils.py

```diff
@@ -416,6 +416,8 @@ def archive_check(archivefile):
         try:
             if archivefile.endswith('.tar.gz'):
                 subprocess.check_call("tar tfz " + archivefile, shell=True)
+            elif archivefile.endswith('.tgz'):
```

Comment:
could jsut merge with above test, tar.gz and tgs are identical, just test on extension (kinda if tag.gz or tgz)
furthermore, below command extracts archive instead of checking archive